css/global.css

```diff
@@ -1,4 +1,6 @@
-/* button:has(div):has(div):has(div):has(div):has(svg), */
-div[data-testid="split-stays"] {
+button:has(div):has(div):has(div):has(div):has(svg[aria-label="Split stay"]),
+div[data-testid="split-stays"],
```

Comment:
This hides the entire split stay section in the list view (separate from the map).